scripts/lib/CIME/XML/files.py

```diff
@@ -32,11 +32,15 @@ def __init__(self):
 
     def get_value(self, vid, attribute=None, resolved=True, subgroup=None):
         value = super(Files, self).get_value(vid, attribute=attribute, resolved=False, subgroup=subgroup)
-        if "COMP_ROOT_DIR" not in vid and value is not None and resolved and "COMP_ROOT_DIR" in value:
-            m = re.search("(COMP_ROOT_DIR_[^/]+)/", value)
-            comp_root_dir_var_name = m.group(1)
-            comp_root_dir = self.get_value(comp_root_dir_var_name, attribute=attribute, resolved=False, subgroup=subgroup)
-            self.set_value(comp_root_dir_var_name, comp_root_dir)
+        if value is None and "COMP_ROOT_DIR" not in vid and attribute is not None:
+            value = super(Files, self).get_value(vid, attribute=None, resolved=False, subgroup=subgroup)
```

Comment:
This is a repeat of an earlier line.

Reply:
No it's not, note the change in attribute.

Reply:
Ah, didn't notice that.